examples/open_llama_4D_benchmark/download_open_llama_ckpt.py

```diff
@@ -1,6 +1,6 @@
 ################################################################################
 #
-# Copyright 2023 ByteDance Ltd. and/or its affiliates. All rights reserved.
+# Copyright 2024 ByteDance Ltd. and/or its affiliates. All rights reserved.
```

Comment:
@MackZackA ？

Reply:
We normally update the year of copyright when made some new changes that year.